ultra-moon-hawlucha/route.mdr

```diff
@@ -0,0 +1,2120 @@
+- Send out Hawlucha
+- HJK + Send out Popplio
+- Revive Zorua + Send out Zorua
+- Revive Hawlucha
+:::
+:::if{source="Hawlucha" condition="$zoruafaintturn == 4"}
+- Send out Hawlucha
+- HJK + Send out Popplio
+- Revive Hawlucha
+:::
+:::if{source="Hawlucha" condition="$zoruafaintturn == 5"}
+- Send out Popplio
+- Pound
+:::
+:::::
+
+:::::if{source="Hawlucha" condition="$necrozmadeath"}
+  :::pokemon[If Zorua dies turn 2:]
+   - Send out Popplio
+   - Pound + Send out Hawlucha
+   - Revive Popplio + Send out Popplio
+   - Revive Zorua + Send out Zorua
+   - Revive Hawlucha
+  :::
+  :::pokemon[If Zorua is alive at turn 3:]
+   - Z-Leer (if Zorua faints this turn, see the guide below this)
+   - Leer + Send out Hawlucha
+   - HJK + Send out Popplio
+   - Revive Hawlucha
+  :::
+  :::pokemon[If Zorua dies turn 3:]
+   - Send out Hawlucha
+   - HJK + Send out Popplio
+   - Revive Zorua + Send out Zorua
+   - Revive Hawlucha
+  :::
+:::::
+
+:info[**Free heal**]{color=pink}
+
+:::if{source="Hawlucha" condition="speed=(7- / x / x)"}
+- Say yes to Poipole.
+- Rearrange Pokemon in this order: Popplio -> Hawlucha -> Poipole -> Zorua
+:::
+:::if{source="Hawlucha" condition="speed=(8+ / x / x)"}
+- Say no to Poipole. (not the end of the world if you mash and hit yes)
+- Rearrange Pokemon in this order: Popplio -> Hawlucha -> Zorua
+:::
+- Move Adrenaline Orb from Popplio to Hawlucha.
+
+:::::trainer[Mina]
+  :::pokemon[Mawile]
+    - Baby-Doll Eyes
+    - Switch to Hawlucha
+    - X Attack
+    - Brick Break
+  :::
+  :::pokemon[Granbull]
+    - Acrobatics
+  :::
+  :::pokemon[Ribombee]
+    - Acrobatics
+  :::
+:::::
+
+- Fly to Hau'oli Cemetery.
+- Switch Hawlucha to the front.
+- If you are in range to die to a Smeargle Extreme Speed:
+  - Heal to full.
+  - Equip Wacan Berry (if you have one) or move Adrenaline Orb to Hawlucha.
+
+::damage[Smeargle's Extreme Speed]{source="Hawlucha" offensive=false movePower=80 stab=true level=60 evs=43 opponentLevel=51 opponentStat=72}
+:::::trainer[Ilima]
+  :::pokemon[Gumshoos]
+    - High Jump Kick
+  :::
+  :::pokemon[Smeargle]
+    - _If not holding an item:_
+      - Acrobatics
+    - Brick Break
+  :::
+  :::pokemon[Komala]
+    - Brick Break
+  :::
+:::::
+
+- Heal to full.
+- If you did not do so already:
+  - Equip Wacan Berry (if you have one) or move Adrenaline Orb to Hawlucha.
+
+:::::trainer[Lana]
+  :::pokemon[Lanturn]
+    - X Attack
+    - (heal Paralysis)
+    - Brick Break
+  :::
+  :::pokemon[Cloyster]
+    - Brick Break
+  :::
+  :::pokemon[Araquanid]
+    - Acrobatics
+  :::
+:::::
+
+- Get the free heal from the NPC to the right of the trial gates.
+- Move Adrenaline Orb to Hawlucha if you equipped Wacan Berry earlier.
+
+:::::trainer[Kiawe]
+  :::pokemon[Arcanine]{info="has Extreme Speed" infoColor=gray}
+    - _If you have 2 X Defenses:_
+      - X Defense
+      - X Attack (x2 if you want to avoid a range)
+      - Acrobatics
+    - _Otherwise:_
+      - X Attack
+      - Acrobatics
```

Comment:
Is the route successfully rendering when you drag it manually? nested lists like this often have issues due to a bug w/ the processor

Reply:
Looks fine to me
![Screenshot_128](https://user-images.githubusercontent.com/104919647/172223000-6e65bc99-e1e9-41af-8815-757e57532c94.png)

Reply:
cool, was just making sure.